optimade/server/logger.py

```diff
@@ -1,9 +1,9 @@
 """Logging to both file and terminal"""
 import logging
+import logging.handlers
```

Comment:
Why do you need to import logging.handlers? I think it will be imported when you import logging.

Reply:
This one isn't a mypy thing, but an annoying Python gotcha to do with automatic imports/global state: https://bugs.python.org/issue42842

Reply:
We were probably getting away with it as actually initializing the loggers before trying to access `logging.handlers` (which should then do the import behind the scenes) but probably better to do it this way to be explicit (hence the warning).